oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
I don't feel strongly about it, so if you do, I'm happy to change it. Ultimately, I think it's kind of meaningless because people can still import it even if it's named with an underscore.

I think underscore-prefixing is a signal to users "you should stay away from messing with this" and I don't think it's necessary to scare users away from some possibly-useful utility functions.

If you don't want to commit to maintaining those functions, don't document them?

If everything internal to the operation of this library is supposed to be 'private', then why isn't `crypt.py` `_crypt.py`? What about `util.py`?